packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/ios/integrationsteps.md

```diff
@@ -1,5 +1,370 @@
+## Integration steps
+
+### Summary
+
+There are many different ways to integrate the Swift SDK into your app. Using
```

Comment:
same comments as before in previous article reviews.

Reply:
Updated. See previous comments.

Reply:
Yup, all good. Resolve.